pydicom/values.py

```diff
@@ -466,14 +466,22 @@ def convert_PN(
     valuerep.PersonName or MultiValue of PersonName
         The decoded 'PN' value(s).
     """
-    def get_valtype(x: bytes) -> PersonName:
-        return PersonName(x, encodings).decode()
 
-    b_split = byte_string.rstrip(b'\x00 ').split(b'\\')
-    if len(b_split) == 1:
-        return get_valtype(b_split[0])
+    encodings = encodings or [default_encoding]
 
-    return MultiValue(get_valtype, b_split)
+    def get_valtype(x: str) -> PersonName:
+        person_name = PersonName(x, encodings)
+        # Using an already decoded string in PersonName constructor leaves
+        # the original string as undefined, let's set it through encode
+        person_name.encode()
+        return person_name.decode()
+
+    stripped_string = byte_string.rstrip(b'\x00 ')
+    decoded_value = decode_bytes(stripped_string, encodings, TEXT_VR_DELIMS)
```

Comment:
I think this could be problematic. I will probably have time for a better look only at the weekend, but at a first glance I would say this would miss the delimiters inside a PN (e.g. `=` and `^`) that reset an encoding, and could produce incorrect text. I may be wrong, though...

Reply:
Ok, I was wrong - the code splits on the escape sequences (could have known that as I wrote it...), so that is not a problem.